cmd/vodloadtester/vodloadtester.go

```diff
@@ -0,0 +1,423 @@
+	TaskCheckSuccess     uint      `json:"taskCheckSuccess,omitempty"`
+	ErrorMessage         string    `json:"errorMessage,omitempty"`
+}
+
+func main() {
+	var cliFlags = &cliArguments{}
+
+	flag.Set("logtostderr", "true")
+	vFlag := flag.Lookup("v")
+
+	fs := flag.NewFlagSet("vodloadtester", flag.ExitOnError)
+
+	fs.IntVar(&cliFlags.Verbosity, "v", 3, "Log verbosity.  {4|5|6}")
+	fs.BoolVar(&cliFlags.Version, "version", false, "Print out the version")
+	fs.BoolVar(&cliFlags.TaskCheck, "task-check", false, "Check task processing")
+
+	fs.BoolVar(&cliFlags.DirectUpload, "direct", false, "Launch direct upload test")
+	fs.BoolVar(&cliFlags.ResumableUpload, "resumable", false, "Launch tus upload test")
+	fs.BoolVar(&cliFlags.Import, "import", false, "Launch import from url test")
+
+	fs.UintVar(&cliFlags.VideoAmount, "video-amt", 1, "How many video to upload")
+	fs.DurationVar(&cliFlags.StartDelayDuration, "delay-between-groups", 10*time.Second, "Delay between starting group of uploads")
+
+	fs.UintVar(&cliFlags.Simultaneous, "sim", 1, "Number of simulteneous videos to upload")
+	fs.StringVar(&cliFlags.Filename, "file", "bbb_sunflower_1080p_30fps_normal_2min.mp4", "File to upload or url to import")
+	fs.StringVar(&cliFlags.APIToken, "api-token", "", "Token of the Livepeer API to be used")
+	fs.StringVar(&cliFlags.APIServer, "api-server", "origin.livepeer.com", "Server of the Livepeer API to be used")
+	fs.StringVar(&cliFlags.OutputPath, "output-path", "/tmp/results.ndjson", "Path to output result .ndjson file")
+
+	_ = fs.String("config", "", "config file (optional)")
+
+	ff.Parse(fs, os.Args[1:],
+		ff.WithConfigFileFlag("config"),
+		ff.WithConfigFileParser(ff.PlainParser),
+		ff.WithEnvVarPrefix("VODLOADTESTER"),
+	)
+	flag.CommandLine.Parse(nil)
+	vFlag.Value.Set(fmt.Sprintf("%d", cliFlags.Verbosity))
+
+	hostName, _ := os.Hostname()
+	runnerInfo := fmt.Sprintf("Hostname %s OS %s", hostName, runtime.GOOS)
+	fmt.Printf("Compiler version: %s %s\n", runtime.Compiler, runtime.Version())
+	fmt.Print(runnerInfo)
+
+	if cliFlags.Version {
+		return
+	}
+
+	if cliFlags.Filename == "" {
+		glog.Fatal("missing --file parameter")
+	}
+
+	var err error
+	var fileName string
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	if cliFlags.APIToken == "" {
+		glog.Fatalf("No API token provided")
+	}
+
+	apiToken := cliFlags.APIToken
+	apiServer := cliFlags.APIServer
+
+	lApiOpts := api.ClientOptions{
+		Server:      apiServer,
+		AccessToken: apiToken,
+		Timeout:     240 * time.Second,
+	}
+	lapi, _ := api.NewAPIClientGeolocated(lApiOpts)
+	vt := &vodLoadTester{
+		lapi:     lapi,
+		ctx:      ctx,
+		cancel:   cancel,
+		cliFlags: *cliFlags,
+	}
+
+	if cliFlags.Import {
+		if cliFlags.DirectUpload || cliFlags.ResumableUpload {
+			glog.Fatal("Cannot use -import with either -direct or -resumable")
+		}
+		fileName = cliFlags.Filename
+		vt.importFromUrlTest(fileName, runnerInfo)
+	}
+
+	if cliFlags.DirectUpload || cliFlags.ResumableUpload {
```

Comment:
Should this be an `else if`, or should we at least `return` from the above branch? From the other checks it seems like it's either one or ther other here.
```suggestion
		vt.importFromUrlTest(fileName, runnerInfo)
		return
	}

	if cliFlags.DirectUpload || cliFlags.ResumableUpload {
```
or
```suggestion
		vt.importFromUrlTest(fileName, runnerInfo)
	} else if cliFlags.DirectUpload || cliFlags.ResumableUpload {
```

Reply:
Yeah, I was doing the check of not mixing the two possible inputs, but it's file like this, I added a return in the import